### ISSUE TITLE
Fail if minimum tox version is not satisfied

### ISSUE THREAD

**Issue by GoogleCodeExporter** [24x-fi/pytox#7]
```
So ... it is possible that newer tox versions may introduce new features (such 
as issue #6 - copying site-packages or hooks) and one's tox.ini may depend on 
it. It would be nice to 'assert' minimum tox version in tox.ini using 
pkg_resources syntax:

$ cat tox.ini
[...]
requires_tox_version = ">=0.8"

So tox can fail if tox.__version__ does not satisfy that requirement.
```

Original issue reported on code.google.com by `Sridhar....@gmail.com` on 12 Aug 2010 at 6:59

**Comment by GoogleCodeExporter** [comment]
```
there now is a "minversion" directive, to be released with the pending 1.0 
release.
```

Original comment by `holger.k...@gmail.com` on 26 May 2011 at 3:48
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 1637a9b59c59.
```

Original comment by `holger.k...@gmail.com` on 26 May 2011 at 3:49
- Changed state: **Fixed**
